### PR TITLE
(python) Add builtins, keyword arguments and constants

### PR DESCRIPTION
**Pull request by wolfy1339**

### CODE REVIEW

**Comment by paladox** [comment]
needs rebase + fix the test.

**Comment by wolfy1339** [comment]
why does it need a rebase? there aren't any merge conflicts.
If a rebase needs to be done, it can be done at merge time

**Comment by paladox** [comment]
Oh i see, then the test is failing.

**Comment by marcoscaceres** [comment]
Hi @wolfy1339, any chance you can check why it’s failing on TravisCI?

File: src/languages/python.js [hunk 1]
```diff
@@ -113,6 +130,14 @@ function(hljs) {
         className: 'meta',
         begin: /^[\t ]*@/, end: /$/
       },
+      {
+        className: 'type',
+        begin: /([\w]+)=[\w"'\.\x20]+[,)]/
+      },
```
Comment (yyyc514) [inline src/languages/python.js]: Please add a `markup` test that shows that both these rules (type and symbol) work as expect and provides and example.

**Comment by yyyc514** [comment]
Evidentally `should markup matrix-multiplication` test is failing also, so you'd need to resolve that.

**Comment by yyyc514** [comment]
@wolfy1339 You still around to champion this?

**Comment by yyyc514** [comment]
Closing, no response and no activity.

**Pull request closed without merging**